src/gui/plugins/apply_force_torque/ApplyForceTorque.hh

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GZ_GUI_APPLYFORCETORQUE_HH_
+#define GZ_GUI_APPLYFORCETORQUE_HH_
+
+#include <memory>
+
+#include <gz/sim/gui/GuiSystem.hh>
+
+namespace gz
+{
+namespace sim
+{
+  class ApplyForceTorquePrivate;
+
+  /// \brief Publish wrench to "/world/apply_link_wrench/wrench" topic.
+  ///
+  /// ## Configuration
+  /// This plugin doesn't accept any custom configuration.
```

Comment:
Knit: Might be worth mentioning which plugin provides the `/world/apply_link_wrench/wrench` topic.